examples/drag-and-drop/.testplane.conf.ts

```diff
@@ -0,0 +1,41 @@
+const SERVER_PORT = process.env.PORT || 3000;
+
+export default {
+    gridUrl: "http://localhost:4444/wd/hub",
+    baseUrl: `http://localhost:${SERVER_PORT}`,
+    pageLoadTimeout: 0,
+    httpTimeout: 60000,
+    testTimeout: 90000,
+    resetCursor: false,
+    sets: {
+        desktop: {
+            files: [
+                "testplane-tests/**/*.testplane.(t|j)s"
+            ],
+            browsers: [
+                "chrome"
+            ]
+        }
+    },
+    browsers: {
+        chrome: {
+            automationProtocol: "devtools",
+            headless: true,
+            desiredCapabilities: {
+                browserName: "chrome"
+            }
+        }
+    },
+    plugins: {
+        "html-reporter/testplane": {
+            // https://github.com/gemini-testing/html-reporter
+            enabled: true,
+            path: "testplane-report",
+            defaultView: "all",
+            diffMode: "3-up-scaled"
+        }
+    },
+    devServer: {
+        command: "npm run server:dev"
+    },
```

Comment:
I suggest to pass SERVER_PORT as environment variable here:
```js
    devServer: {
        command: "npm run server:dev",
        env: { PORT: SERVER_PORT },
    },
```
instead of passing it via npm script

Reply:
And i think you could use `npx http-server ./static/` as command

Reply:
in this case you will not be able to run the dev-server separately with the same port (8080 by default)